_activities/forcesfriction.md

```diff
@@ -5,12 +5,27 @@ lang: en
 # Content
 #
 title: "Forces and Friction"
-title_fr:
+title_fr:"Forces et Friction"
 age: 8-12
 objective: "Learn how forces affects the motion of objects."
-objective_fr:
-summary:
-summary_fr:
+objective_fr:"Apprendre comment que les forces influencent le mouvement d'un objet."
+summary:"In this activity, students will perform experiments using a simple pulley-weight setup in order to understand the effects of forces on the motion of an object. They will learn about how mass affects the motion of the object by adding washers to both ends of a string that is attached to a pulley.  They will also experiment with the force of friction acting on the object, and discover how a sandpaper surface affects the object’s motion."
```

Comment:
Looks pretty good! I would remove "simple" as an adjective of the setup (in general it's best not to make assumptions on what the reader will think is simple.) Also, please change "how mass affects the motion of the object" to "how the force applied on the object and the mass of the object affect its motion". Change "both ends of a string that is attached to a pulley" to "change the weight of the object or the strength of the force". And equivalent changes in French.